rest/tests/test_rest.py

```diff
@@ -1,9 +1,12 @@
 """
 Unit tests for REST api
 """
-
+# pylint: disable=too-many-lines
```

Comment:
Is there a good reason not to split the file up? Do we, as a group, care about `too-many-lines`? It seems like a good thing to adhere to.